apps/mobile/src/app/wallet/(tabs)/_layout.tsx

```diff
@@ -187,7 +218,7 @@ export default function TabLayout() {
           }}
         />
       </Tabs>
-      {createFilledActionBar(ref, router)()}
+      {CreateFilledActionBar(ref, router)()}
```

Comment:
I see this pattern was here from before, but very odd imo and a react anti-pattern, unless there's some RN quirk I'm not aware of

Reply:
![image](https://github.com/user-attachments/assets/40cbad97-0056-41e4-8ae7-61805e0193f3)